src/OrchardCore.Modules/OrchardCore.AdminMenu/Recipes/AdminMenuStep.cs

```diff
@@ -33,8 +33,14 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
             foreach (JObject token in model.Data)
             {
                 var adminMenu = token.ToObject<Models.AdminMenu>(serializer);
-                adminMenu.Id = Guid.NewGuid().ToString("n");// we always add it as a new tree.
-                await _AdminMenuService.SaveAsync(adminMenu);
+
+                // When the id is not supplied create a new menu, otherwise replace the existing menu, or create a new menu.
```

Comment:
Rephrase comment/add comma: 2 times 'create a new menu' at the beginning and at the end.

Reply:
Better?